src/runtime-rs/crates/hypervisor/src/device/block.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#[derive(Debug)]
+pub struct BlockConfig {
+    /// Unique identifier of the drive.
+    pub id: String,
+
+    /// Path of the drive.
+    pub path_on_host: String,
+
+    /// If set to true, the drive is opened in read-only mode. Otherwise, the
+    /// drive is opened as read-write.
+    pub is_read_only: bool,
```

Comment:
`is_read_only` or `is_readonly`, which is better?

Reply:
Thanks for your review, change to use `is_readonly`.